DependencyInjection/KnpGaufretteExtension.php

```diff
@@ -72,6 +73,11 @@ private function createAdapter($name, array $config, ContainerBuilder $container
         foreach ($config as $key => $adapter) {
             if (array_key_exists($key, $factories)) {
                 $id = sprintf('gaufrette.%s_adapter', $name);
+                // If version of Symfony equal or more than 3.2, it might have an runtime environment variables,
+                // so we need process them also, otherwize it provides wrong parameters
+                if (Kernel::MAJOR_VERSION >= 3 && Kernel::MINOR_VERSION >= 2) {
```

Comment:
Be careful, this condition won't pass for `4.0`, `4.1`, `5.0`, `5.1`, etc...

Reply:
It can be replaced by this condition below to match all Symfony's versions equal or superior to `3.2.0` : 

```php
Kernel::VERSION_ID >= 30200
```

Reply:
Thx, I updated condition